Core/src/com/serotonin/timer/SimulationTimer.java

```diff
@@ -70,6 +72,23 @@ public void next() {
         fastForwardTo(timeSource.currentTimeMillis() + 1);
     }
 
+    /**
+     * @param duration duration to increase the clock by (relative to clock's current instant)
+     * @return the current instant (same as {@link #instant() would return}
+     */
+    public Instant fastForward(Duration duration) {
+        return fastForwardTo(instant().plus(duration));
+    }
+
+    /**
+     * @param time the new instant to set the clock to
+     * @return the current instant (same as {@link #instant() would return}
+     */
+    public Instant fastForwardTo(Instant time) {
+        fastForwardTo(time.toEpochMilli());
+        return time;
+    }
```

Comment:
New API in a patch release, remove this, it's not required.

Reply:
Ok, removing it.